src/compiler/checker.ts

```diff
@@ -28997,7 +28999,7 @@ namespace ts {
                     }
                     return isErrorType(apparentType) ? errorType : apparentType;;

```

Comment:
`;;` :(

Reply:
I sent a quick fix for this.

Reply:
I don’t recall seeing this or touching this line. Weird.

Reply:
Oh, it’s because you did